pkgs/applications/networking/appgate-sdp/default.nix

```diff
@@ -0,0 +1,135 @@
+{ alsaLib
+, atk
+, cairo
+, cups
+, curl
+, dbus
+, dpkg
+, expat
+, fetchurl
+, fontconfig
+, freetype
+, gdk-pixbuf
+, glib
+, gnome2
+, gtk3
+, krb5
+, lib
+, libX11
+, libxcb
+, libXScrnSaver
+, libXcomposite
+, libXcursor
+, libXdamage
+, libXext
+, libXfixes
+, libXi
+, libXrandr
+, libXrender
+, libXtst
+, libnotify
+, libuuid
+, lttng-ust
+, libsecret
+, nspr
+, nss
+, pango
+, python37
+, python37Packages
+, stdenv
+, systemd
+, at-spi2-atk
+, at-spi2-core
+, autoPatchelfHook
+, makeWrapper
+, zlib
+, icu
+, openssl
+, coreutils
+, procps
+, e2fsprogs
+, iproute
+, dnsmasq
+, libappindicator-gtk3
+}:
+with stdenv.lib;
+let
+  deps = [ alsaLib at-spi2-atk at-spi2-core atk cairo cups curl dbus expat gdk-pixbuf glib gtk3 libuuid lttng-ust libsecret nspr nss pango python37 python37Packages.dbus-python libX11 libXScrnSaver libXcomposite libXcursor libXdamage libXext libXfixes libXi libXrandr libXrender libXtst libxcb stdenv.cc.cc libappindicator-gtk3 ];
+  rpath = makeLibraryPath deps + ":${stdenv.cc.cc.lib}/lib64";
+in
+stdenv.mkDerivation rec {
+  name = "appgate-sdp-${version}";
```

Comment:
```suggestion
  pname = "appgate-sdp";
```

Reply:
I would love to use `autoPatchlfHook` instead, but it doesn't work for this particular package. After it succeeded I see some libraries with missing rpath, which prevents app from starting. I did refine the whole `patchelf` and simplified it.